hasjob/views/listing.py

```diff
@@ -583,9 +583,21 @@ def rejectjob(domain, hashid):
                 post.domain.banned_reason = rejectform.reason.data
                 if post.user:
                     post.user.blocked = True
+
+                # In case of Ban, Reject all the job posts by the same domain
+                # unless it's a webmail domain
+                if not post.domain.is_webmail:
+                    for jobpost in post.domain.jobposts:
+                        if jobpost.is_listed():
+                            jobpost.status = POSTSTATUS.REJECTED
+                            jobpost.closed_datetime = datetime.utcnow()
+                            jobpost.review_comments = rejectform.reason.data
+                            jobpost.review_datetime = datetime.utcnow()
```

Comment:
Make these `db.func.utcnow()`. We've introduced this method recently and are switching over everywhere. It pushes timestamp generation into the database, where it is transaction-bound, so all calls to `utcnow()` within the transaction will return the same value.